lib/msf/core/modules/metadata/obj.rb

```diff
@@ -28,8 +28,10 @@ class Obj
   attr_reader :description
   # @return [Array<String>]
   attr_reader :references
-  # @return [Boolean]
+  # @return [String]
   attr_reader :platform
+  # @return [Msf::Module::PlatformList]
+  attr_reader :platform_list
```

Comment:
```suggestion
  attr_reader :platforms
```

Reply:
you can also call `platforms` on a `PlatformList` so calling it platforms leads to `platforms.platforms` which I didn't like either 😅